openai/cli.py

```diff
@@ -224,6 +227,41 @@ def list(cls, args):
 
 
 class Search:
+    @classmethod
+    def prepare_data(cls, args):
```

Comment:
Doc comment saying what this method does? It's confusing. I have no idea what the inputs or outputs are, or what it does, without reverse engineering.